build/css-selector-generator.js

```diff
@@ -4,7 +4,14 @@
 
   CssSelectorGenerator = (function() {
     CssSelectorGenerator.prototype.default_options = {
-      selectors: ['id', 'class', 'tag', 'nthchild']
+      selectors: ['id', 'class', 'tag', 'nthchild'],
+      prefix_tag: false,
+      log: false,
```

Comment:
It seems like this option is unused, unless I have bad eyes :).

Reply:
Just used for guarding console.log while I am testing. Prefer to leave it there, so that in a particular test we can set it to true and debug.